README.md

```diff
@@ -2,6 +2,8 @@
 
 **Spotify has disabled my API access for the skill, it was violating their Terms of Service by enabling voice control. I must have missed this back in 2017 when I created the skill.**
 
+To use this skill currently you will need to create your own application using [The Spotify developer dashboard](https://developer.spotify.com/dashboard/). See instructions below under Personal Access Token.
```

Comment:
(we MIGHT want to say "due to Spotify EULA" and perhaps add a bottom section "Appendix" explaining the current state?  that is: hopefully in the future Spotify will accept that this isn't a "commercial" use, and a default client APP from this open source integration would be accepted)

Reply:
Added something along these lines.